jupyterhub/app.py

```diff
@@ -2174,6 +2174,7 @@ def _log_cls(name, cls):
             os.remove(self.pid_file)
 
         # finally stop the loop once we are all cleaned up
+        asyncio.get_event_loop().stop()
```

Comment:
I don't believe stop should be part of cleanup. That should be handled at a separate level, where cleanup is called.

Reply:
Done